README.md

```diff
@@ -1,108 +1,109 @@
-   $ chmod +x bazel-0.16.0-installer-linux-x86_64.sh
-   $ ./bazel-0.16.0-installer-linux-x86_64.sh --user
-   ```
+        wget https://github.com/bazelbuild/bazel/releases/download/0.16.0/bazel-0.16.0-installer-linux-x86_64.sh      
+        chmod +x bazel-0.16.0-installer-linux-x86_64.sh
+        ./bazel-0.16.0-installer-linux-x86_64.sh --user
 
 2. Add and source the ``bin`` path to your ``~/.bashrc`` file in order to be 
    able to call bazel from the user's installation we set up:
 
-   ```  
-   export PATH=$PATH:~/bin
-   $ source ~/.bashrc   
-   ```
+        export PATH=$PATH:~/bin
+        source ~/.bashrc   
 
-3. Ensure that all the TensorFlow dependencies are installed, as per the
-   TensorFlow [prepare environment] for linux.
+3. Ensure that all the TensorFlow dependencies are installed, and when building 
+   TensorFlow* *do not* select `Yes` when asked: 
+        Do you wish to build TensorFlow with CUDA support? [y/N]: N
+        No CUDA support will be enabled for TensorFlow.
 
-   **Note:** You do not need CUDA in order to use the ngraph-tensorflow bridge.
 
 4. Additional dependencies.
    - Install ```apt-get install libicu-dev``` to avoid the following (potential) error:
      ```unicode/ucnv.h: No such file or directory```.
 
 
-### Installation
+#### Installation
 
 1. Once TensorFlow's dependencies are installed, clone the source of the 
    [tensorflow] repo to your machine. 
 
-   :warning: You need the following version of TensorFlow: `v1.10.0`
-
-   ```
-   $ git clone https://github.com/tensorflow/tensorflow.git
-   $ cd tensorflow
-   $ git checkout v1.10.0
-   $ git status
-   HEAD detached at v1.10.0
-   ```
-2. When setting up and activating the virtual environment with TensorFlow 
-   frameworks, you must use a specific kind of venv designed for 
-   ``system-site-packages``.  A regular venv tends to not detect nGraph installs:
-
-   ```
-   virtualenv --system-site-packages -p /usr/bin/python2 <your_virtual_env_dir> # for Python 2.7
-   ```
-   For Python 3.n version:
-   ```
-   virtualenv --system-site-packages -p python3 <your_virtual_env_dir> # for Python 3.n
-   ```
-   Activate virtual environment:
-   ```
-   source <your_virtual_env_dir>/bin/activate # bash, sh, ksh, or zsh
-   ```
-
+     :warning: You need the following version of TensorFlow: `v1.10.0`
+
+        git clone https://github.com/tensorflow/tensorflow.git
+        cd tensorflow
+        git checkout v1.10.0
+        git status
+        HEAD detached at v1.10.0
+   
+2. You must instantiate a specific kind of `virtualenv`  to be able to proceed 
+   with the `ngraph-tf` bridge installation. For systems with Python 3.n or 
+   Python 2.7, these commands are
+
+        virtualenv --system-site-packages -p python3 your_virtualenv 
+        virtualenv --system-site-packages -p /usr/bin/python2 your_virtualenv  
+        source your_virtualenv/bin/activate # bash, sh, ksh, or zsh
```

Comment:
Venv instructions repeat. Maybe refactor them out of option 1 and 2 and add a "create venv before proceeding (or install in your system python, but venv is recommended)" note at the beginning?